src/fundus/publishers/de/__init__.py

```diff
@@ -299,7 +299,11 @@ class DE(PublisherEnum):
         domain="https://www.businessinsider.de/",
         sources=[
```

Comment:
While we're at it, I also found a RSS Feed: https://www.businessinsider.de/feed/businessinsider-alle-artikel